test/controllers/tasks/statement.of.capital.controller.unit.ts

```diff
@@ -43,15 +52,33 @@ describe("Statement of Capital controller tests", () => {
     mocks.mockSessionMiddleware.mockClear();
     mockGetStatementOfCapitalData.mockClear();
     mockUpdateConfirmationStatement.mockClear();
+    mockGetShareholders.mockClear();
+    mockSendUpdate.mockClear();
   });
 
   describe("get tests", () => {
-    it("should navigate to the statement of capital page", async () => {
+    it("should navigate to the statement of capital page with NO buttons visible if share totals DO NOT match the shareholders'.", async () => {
       mockGetStatementOfCapitalData.mockReturnValueOnce(mockStatementOfCapital);
+      mockGetShareholders.mockReturnValue([{ shares: "123" }]);
```

Comment:
get shareholders and sendUpdate are async functions so best to use mockResolvedValueOnce instead of mockReturnValueOnce